packages/zipkin/src/option.js

```diff
@@ -48,24 +49,13 @@ class Some {
     return other instanceof Some && other.value === this.value;
   }
   toString() {
-    return `Some(${this.value.toString()})`;
-  }
-  get present() {
-    return true;
-  }
-  get type() {
-    return 'Some';
+    return `Some(${this.value})`;
```

Comment:
I removed `.toString()` in a few places.  Letting template strings handle it seems safer.  Since Some is parametric, it's valid to have a Some whose value is `null` or `undefined`, which would cause `.toString()` to throw.  Template strings will handle `null` and `undefined` gracefully.